R/package.R

```diff
@@ -107,7 +107,11 @@ xml_parse_data <- function(x, includeText = NA, pretty = FALSE) {
   }
 
   ## Order the nodes properly
-  pd <- pd[ order(pd$line1, pd$col1, -pd$line2, -pd$col2, pd$terminal), ]
+  ## - the terminal nodes from pd2 may be nested inside each other, when
+  ##   this happens they will have the same line1, col1, line2, col2 and
+  ##   terminal status; and 'start' is used to break ties
+  ord <- with(pd, order(line1, col1, -line2, -col2, terminal, -start))
```

Comment:
In general using `with()` in packages is not recommended, mainly because it will generate a NOTE about global variables, which you then have to whitelist with `utils::globalVariables()`. I think you should convert this to

```r
pd <- pd[ order(pd$line1, pd$col1, -pd$line2, -pd$col2, pd$terminal, -start), ]
```

Reply:
Will do (but will use `-pd$start`)